tpgtools/overrides/logging/samples/logview/basic.tf.tmpl

```diff
@@ -0,0 +1,13 @@
+resource "google_logging_log_view" "primary" {
+  name        = "{{view}}"
+  bucket      = google_logging_project_bucket_config.basic.id
+  description = "A logging view configured with Terraform"
+  filter      = "SOURCE(\"projects/myproject\") AND resource.type = \"gce_instance\" AND LOG_ID(\"stdout\")"
+}
+
+resource "google_logging_project_bucket_config" "basic" {
+    project        = "{{project}}"
+    location       = "global"
+    retention_days = 30
+    bucket_id      = "_Default"
```

Comment:
just curious, does this value have significance? Why `_Default`?

Reply:
It does. `_Default` and `_Required` are the two buckets which already exist in every project, organization, folder, and billing account. The acceptance test acquires an existing bucket rather than create a new one.